packages/manager/apps/dedicated/client/app/dedicated/server/monitoring/update/dedicated-server-monitoring-update.controller.js

```diff
@@ -39,6 +39,7 @@ angular.module('App').controller(
       return this.Server.updateMonitoring(
         this.$stateParams.productId,
         !monitored,
+        !monitored, // reset no-intervention to monitored value
```

Comment:
Looks like the same parameter is passed twice

Reply:
yes but for two different arguments, only in this case. The second parameter is for "noIntervention". This needs to be reset to "minitored" value whenever it changes. For example, if "minitored" is set to true, "noIntervention" needs to be set to true and vice versa.